src/Microsoft.AspNetCore.Diagnostics.Abstractions/IExceptionHandlerFeature.cs

```diff
@@ -8,5 +8,7 @@ namespace Microsoft.AspNetCore.Diagnostics
     public interface IExceptionHandlerFeature
     {
         Exception Error { get; }
+
+        string Path { get; }
```

Comment:
Yes it's breaking, but is it an acceptable break? There's no expectation that anyone implement this interface, and even a github search only shows consumers.